include/maths/CBoostedTreeImpl.h

```diff
@@ -662,9 +676,8 @@ class MATHS_EXPORT CBoostedTreeImpl final {
 
     //! Compute the sum loss for the predictions from \p frame and the leaf
     //! count and squared weight sum from \p forest.
-    TDoubleDoubleDoubleTr regularisedLoss(const core::CDataFrame& frame,
-                                          const core::CPackedBitVector& trainingRowMask,
-                                          const TNodeVecVec& forest) const;
+    TDoubleDoublePr gainAndCurvatureAtPercentile(double percentile,
```

Comment:
Function documentation has to be updated.

Reply:
[e0dd26e](https://github.com/elastic/ml-cpp/pull/686/commits/e0dd26ede432e16cd19d7ce0961f925db920f2ef)